scss/detail.scss

```diff
@@ -0,0 +1,39 @@
+$gray : #828282;
+
+.book-container {
+  display: flex;
+  font-family: 'Roboto', sans-serif;
+  padding: 15px;
+}
+
+.column {
+  flex-direction: column;
```

Comment:
Add `display: flex`, otherwise it won't work.

Reply:
Done